kuma/wiki/tests/test_content.py

```diff
@@ -988,109 +987,115 @@ def test_editor_safety_filter(self):
         assert normalize_html(expected_src) == normalize_html(result_src)
 
 
-class AllowedHTMLTests(KumaTestCase):
-    simple_tags = (
-        'div', 'span', 'p', 'h1', 'h2', 'h3', 'h4', 'h5', 'pre',
-        'code', 'dl', 'dt', 'dd', 'table',
-        'section', 'header', 'footer',
-        'nav', 'article', 'aside', 'figure', 'dialog', 'hgroup',
-        'mark', 'time', 'meter', 'output', 'progress',
-        'audio', 'details', 'datagrid', 'datalist', 'table',
-        'address'
-    )
+@pytest.mark.parametrize(
+    'tag',
+    # Sample of tags from ALLOWED_TAGS
+    ('address',
+     'article',
+     'code',
+     'datagrid',
+     'details',
+     'dt',
+     'figure',
+     'h5',
+     'mark',
+     'output',
+     'pre',
+     'progress',
+     ))
+def test_clean_content_allows_simple_tag(tag):
+    """clean_content allows simple tags, id attribute."""
+    html = '<%(tag)s id="foo"></%(tag)s>' % {'tag': tag}
+    assert clean_content(html) == html
+
+
+@pytest.mark.parametrize(
+    'tag',
+    ('br',
+     'command',
+     'img',
+     'input',
+     ))
+def test_clean_content_allows_self_closed_tags(tag):
+    """clean_content allows self-closed tags."""
+    html = '<%s>' % tag
+    assert clean_content(html) == html
+
+
+def test_clean_content_preserves_whitespace():
+    """clean_content allows an HTML table."""
+    html = ('<table><thead><tr><th>foo</th></tr></thead>'
+            '<tbody><tr><td>foo</td></tr></tbody></table>')
+    assert clean_content(html) == html
+
+
+@pytest.mark.parametrize(
+    'html',
+    ('<command id="foo">',
+     '<img align="left" alt="picture of foo" class="foo" dir="rtl" id="foo" src="foo" title="foo">',
+     '<a class="foo" href="foo" id="foo" title="foo">foo</a>',
+     '<div class="foo">foo</div>',
+     '<video class="movie" controls id="some-movie" lang="en-US" src="some-movie.mpg">Fallback</video>',
+     ))
+def test_clean_content_allows_some_attributes(html):
+    """clean_content allows attributes, but doesn't preserve order."""
+    assert normalize_html(clean_content(html)) == html
+
+
+def test_clean_content_allows_some_some_styles():
```

Comment:
Seems like a typo in the name? Probably wanted `test_clean_content_allows_some_styles` instead?